### PR TITLE
Fix hedera local not starting and not generating accounts

### PR DESCRIPTION
**Pull request by natanasow**
Signed-off-by: nikolay <n.atanasow94@gmail.com>

**Description**:

Found out two general errors that prevent hedera-local from starting up the containers
- some platforms don't support the "experimental garbage collector" defined in .env `PLATFORM_JAVA_OPTS`
- if there are dangling docker networks

**Related issue(s)**:

Fixes #
- https://github.com/hashgraph/hedera-local-node/issues/106
- https://github.com/hashgraph/hedera-local-node/issues/76
- Update docs

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -10,7 +10,7 @@ HAVEGED_IMAGE_TAG=0.27.4
 #### Java Process Settings ####
 PLATFORM_JAVA_HEAP_MIN=256m
 PLATFORM_JAVA_HEAP_MAX=2g
-PLATFORM_JAVA_OPTS="-XX:+UnlockExperimentalVMOptions -XX:+UseZGC -Xlog:gc*:gc.log"
+PLATFORM_JAVA_OPTS=
```
Comment (nathanklick) [inline .env:13]: Linux amd64, Linux arm64, MacOS am64, and MacOS arm64, and Windows amd64 all support ZGC. The JVM runs in a Linux container. There is no need to change these settings. Changing the garabage collector may have undesirable results.

What system or environment had an issue eith ZGC?

Comment (natanasow) [reply .env:13]: I'm using `Ubuntu 20.04.3 LTS` and the error I got is
```
network-node                 | Error: Could not find or load main class "-XX:+UnlockExperimentalVMOptions
network-node                 | Caused by: java.lang.ClassNotFoundException: "-XX:+UnlockExperimentalVMOptions
network-node exited with code 1
```

Comment (nathanklick) [reply .env:13]: @natanasow Would you please post the output from `docker version` on your machine?

Comment (nathanklick) [reply .env:13]: Also please post the output from `docker-ompose --version`

Comment (natanasow) [reply .env:13]: - docker version
```
$ docker version
Client: Docker Engine - Community
 Version:           20.10.12
 API version:       1.41
 Go version:        go1.16.12
 Git commit:        e91ed57
 Built:             Mon Dec 13 11:45:33 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.12
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.12
  Git commit:       459d0df
  Built:            Mon Dec 13 11:43:42 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.13
  GitCommit:        9cc61520f4cd876b86e77edfeb88fbcd536d1f9d
 runc:
  Version:          1.0.3
  GitCommit:        v1.0.3-0-gf46b6ba
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```
- docker-compose --version
```
$ docker-compose --version
docker-compose version 1.25.0
```
- docker inspect network-node
```$ docker inspect network-node
[
    {
        "Id": "0d73d5dee15f553fb460f952374fe14e7409e653ce7e0f2e85a26a8e233fdca9",
        "Created": "2022-07-14T15:42:43.380177111Z",
        "Path": "/opt/hgcapp/services-hedera/HapiApp2.0/entrypoint.sh",
        "Args": [],
        "State": {
            "Status": "exited",
            "Running": false,
            "Paused": false,
            "Restarting": false,
            "OOMKilled": false,
            "Dead": false,
            "Pid": 0,
            "ExitCode": 1,
            "Error": "",
            "StartedAt": "2022-07-14T15:42:44.065602779Z",
            "FinishedAt": "2022-07-14T15:42:44.126172709Z",
            "Health": {
                "Status": "unhealthy",
                "FailingStreak": 0,
                "Log": []
            }
        },
        "Image": "sha256:467ff5ceb5190390f91f6d221ce9a607b825cef8d7c803cbea2bf8fa167d6ffd",
        "ResolvConfPath": "/var/lib/docker/containers/0d73d5dee15f553fb460f952374fe14e7409e653ce7e0f2e85a26a8e233fdca9/resolv.conf",
        "HostnamePath": "/var/lib/docker/containers/0d73d5dee15f553fb460f952374fe14e7409e653ce7e0f2e85a26a8e233fdca9/hostname",
        "HostsPath": "/var/lib/docker/containers/0d73d5dee15f553fb460f952374fe14e7409e653ce7e0f2e85a26a8e233fdca9/hosts",
        "LogPath": "/var/lib/docker/containers/0d73d5dee15f553fb460f952374fe14e7409e653ce7e0f2e85a26a8e233fdca9/0d73d5dee15f553fb460f952374fe14e7409e653ce7e0f2e85a26a8e233fdca9-json.log",
        "Name": "/network-node",
        "RestartCount": 0,
        "Driver": "overlay2",
        "Platform": "linux",
        "MountLabel": "",
        "ProcessLabel": "",
        "AppArmorProfile": "docker-default",
        "ExecIDs": null,
        "HostConfig": {
            "Binds": [
                "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/hedera.key:/opt/hgcapp/services-hedera/HapiApp2.0/hedera.key:rw",
                "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/data/config:/opt/hgcapp/services-hedera/HapiApp2.0/data/config:rw",
                "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/network-logs/node/recordStreams:/opt/hgcapp/recordStreams:rw",
                "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/data/onboard:/opt/hgcapp/services-hedera/HapiApp2.0/data/onboard:rw",
                "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/log4j2.xml:/opt/hgcapp/services-hedera/HapiApp2.0/log4j2.xml:rw",
                "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/config.txt:/opt/hgcapp/services-hedera/HapiApp2.0/config.txt:rw",
                "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/data/keys:/opt/hgcapp/services-hedera/HapiApp2.0/data/keys:rw",
                "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/network-logs/node/logs:/opt/hgcapp/services-hedera/HapiApp2.0/output:rw",
                "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/network-logs/node/accountBalances:/opt/hgcapp/accountBalances:rw",
                "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/network-logs/node/stats:/opt/hgcapp/services-hedera/HapiApp2.0/data/stats:rw",
                "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/hedera.crt:/opt/hgcapp/services-hedera/HapiApp2.0/hedera.crt:rw",
                "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/settings.txt:/opt/hgcapp/services-hedera/HapiApp2.0/settings.txt:rw"
            ],
            "ContainerIDFile": "",
            "LogConfig": {
                "Type": "json-file",
                "Config": {}
            },
            "NetworkMode": "network-node-bridge",
            "PortBindings": {
                "50211/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "50211"
                    }
                ],
                "50212/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "50212"
                    }
                ]
            },
            "RestartPolicy": {
                "Name": "",
                "MaximumRetryCount": 0
            },
            "AutoRemove": false,
            "VolumeDriver": "",
            "VolumesFrom": [],
            "CapAdd": null,
            "CapDrop": null,
            "CgroupnsMode": "host",
            "Dns": null,
            "DnsOptions": null,
            "DnsSearch": null,
            "ExtraHosts": null,
            "GroupAdd": null,
            "IpcMode": "shareable",
            "Cgroup": "",
            "Links": null,
            "OomScoreAdj": 0,
            "PidMode": "",
            "Privileged": false,
            "PublishAllPorts": false,
            "ReadonlyRootfs": false,
            "SecurityOpt": null,
            "UTSMode": "",
            "UsernsMode": "",
            "ShmSize": 67108864,
            "Runtime": "runc",
            "ConsoleSize": [
                0,
                0
            ],
            "Isolation": "",
            "CpuShares": 0,
            "Memory": 8589934592,
            "NanoCpus": 0,
            "CgroupParent": "",
            "BlkioWeight": 0,
            "BlkioWeightDevice": null,
            "BlkioDeviceReadBps": null,
            "BlkioDeviceWriteBps": null,
            "BlkioDeviceReadIOps": null,
            "BlkioDeviceWriteIOps": null,
            "CpuPeriod": 0,
            "CpuQuota": 0,
            "CpuRealtimePeriod": 0,
            "CpuRealtimeRuntime": 0,
            "CpusetCpus": "",
            "CpusetMems": "",
            "Devices": null,
            "DeviceCgroupRules": null,
            "DeviceRequests": null,
            "KernelMemory": 0,
            "KernelMemoryTCP": 0,
            "MemoryReservation": 0,
            "MemorySwap": 8589934592,
            "MemorySwappiness": 0,
            "OomKillDisable": false,
            "PidsLimit": null,
            "Ulimits": null,
            "CpuCount": 0,
            "CpuPercent": 0,
            "IOMaximumIOps": 0,
            "IOMaximumBandwidth": 0,
            "MaskedPaths": [
                "/proc/asound",
                "/proc/acpi",
                "/proc/kcore",
                "/proc/keys",
                "/proc/latency_stats",
                "/proc/timer_list",
                "/proc/timer_stats",
                "/proc/sched_debug",
                "/proc/scsi",
                "/sys/firmware"
            ],
            "ReadonlyPaths": [
                "/proc/bus",
                "/proc/fs",
                "/proc/irq",
                "/proc/sys",
                "/proc/sysrq-trigger"
            ]
        },
        "GraphDriver": {
            "Data": {
                "LowerDir": "/var/lib/docker/overlay2/84cd7000f53bfa802b69e0a4adc62a38b575aeba3c9632bea02335854993d8e4-init/diff:/var/lib/docker/overlay2/34da375a33a4dbb7ee8cba917e76ae822c382131ac8d133de604751c4ba41583/diff:/var/lib/docker/overlay2/5d7452116c703ce1d9c99867af6c0e83e772c47858c70eeaee7b00d65291d5a1/diff:/var/lib/docker/overlay2/cd324da2c9641329048e2eb1b519f436ea216894e7dd0cc7f19f14357bc07c20/diff:/var/lib/docker/overlay2/4e36acecb949a506f920598c2be2ca591544ebd1716ed3cae93fe6b7e58a8d2b/diff:/var/lib/docker/overlay2/13b362f679876cc71932ae88436e6ff6da887651459984d17564f6e8bf746a50/diff:/var/lib/docker/overlay2/a0238dff36ed06365eb35b9ab47ab83670d0fea4af1cb743e5252ae1d436e583/diff:/var/lib/docker/overlay2/df36522bbc5a24039bd442787f0abd3ded6ef9e45cc3fd4e867980fa1cd7deb2/diff:/var/lib/docker/overlay2/7d36a293d1d0e0a5c42d62a31dff402788e38f45a72c3c8496b8ccf0b265d7df/diff:/var/lib/docker/overlay2/f3a7bd0b9ecd82319d56aadfcc25e62fd3424f09d218efb9bdd389213b48a192/diff:/var/lib/docker/overlay2/614b229be198e4a47336b1ebea666dd1e96cce2848b45c7cb798f75f58e76d82/diff:/var/lib/docker/overlay2/96b9d30928a35418759b80528534e0d25ed966a8af62f13bd638a911869d9fff/diff:/var/lib/docker/overlay2/21d48d7133cf5d86da309cfb967746f5400e91da3ddb8ae6c7cdaaa8961dadb7/diff:/var/lib/docker/overlay2/96d6bb66545514742f82b84f7771fb79ede1375e2204e419d1f11019400bb1b6/diff:/var/lib/docker/overlay2/972c99c10cd100c4ae876a5bbbc1ad3d5f0a617633fc1f8f03d7f03e36fc42fd/diff:/var/lib/docker/overlay2/c7f9af601d8a1246a2a44a06a6aa9f411cb061a792ab90fb2bfe9d121b7cf90c/diff:/var/lib/docker/overlay2/825b1921460a6beb2f65922910c06ad9cca56576b9b2b976cdbec379353a16c1/diff:/var/lib/docker/overlay2/2d28691565a77dcebad7b532a4bad189cd61b4167ea802b67946dc243118026e/diff:/var/lib/docker/overlay2/78dd2bb52bd05d70ea1c70f12b97b257bd1e4463802ae9888721cd0bd2a16838/diff",
                "MergedDir": "/var/lib/docker/overlay2/84cd7000f53bfa802b69e0a4adc62a38b575aeba3c9632bea02335854993d8e4/merged",
                "UpperDir": "/var/lib/docker/overlay2/84cd7000f53bfa802b69e0a4adc62a38b575aeba3c9632bea02335854993d8e4/diff",
                "WorkDir": "/var/lib/docker/overlay2/84cd7000f53bfa802b69e0a4adc62a38b575aeba3c9632bea02335854993d8e4/work"
            },
            "Name": "overlay2"
        },
        "Mounts": [
            {
                "Type": "bind",
                "Source": "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/network-logs/node/stats",
                "Destination": "/opt/hgcapp/services-hedera/HapiApp2.0/data/stats",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/hedera.crt",
                "Destination": "/opt/hgcapp/services-hedera/HapiApp2.0/hedera.crt",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "volume",
                "Name": "1942425ef3d93bab204b0154160bffa0800b24671a3d24354fc20767e319b160",
                "Source": "/var/lib/docker/volumes/1942425ef3d93bab204b0154160bffa0800b24671a3d24354fc20767e319b160/_data",
                "Destination": "/opt/hgcapp/services-hedera/HapiApp2.0/data/saved",
                "Driver": "local",
                "Mode": "",
                "RW": true,
                "Propagation": ""
            },
            {
                "Type": "bind",
                "Source": "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/data/config",
                "Destination": "/opt/hgcapp/services-hedera/HapiApp2.0/data/config",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/log4j2.xml",
                "Destination": "/opt/hgcapp/services-hedera/HapiApp2.0/log4j2.xml",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/network-logs/node/logs",
                "Destination": "/opt/hgcapp/services-hedera/HapiApp2.0/output",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/network-logs/node/recordStreams",
                "Destination": "/opt/hgcapp/recordStreams",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/data/keys",
                "Destination": "/opt/hgcapp/services-hedera/HapiApp2.0/data/keys",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "volume",
                "Name": "c2169c8202eadb14317d3975af5e580d88eb58189d488643c88f8ed11fff06b7",
                "Source": "/var/lib/docker/volumes/c2169c8202eadb14317d3975af5e580d88eb58189d488643c88f8ed11fff06b7/_data",
                "Destination": "/opt/hgcapp/eventsStreams",
                "Driver": "local",
                "Mode": "",
                "RW": true,
                "Propagation": ""
            },
            {
                "Type": "volume",
                "Name": "bac6ec4dcf32f8258e83ebd4b7c5acb49b181a66333eb15f1fe70b9641e420c1",
                "Source": "/var/lib/docker/volumes/bac6ec4dcf32f8258e83ebd4b7c5acb49b181a66333eb15f1fe70b9641e420c1/_data",
                "Destination": "/opt/hgcapp/services-hedera/HapiApp2.0/data/diskFs",
                "Driver": "local",
                "Mode": "",
                "RW": true,
                "Propagation": ""
            },
            {
                "Type": "volume",
                "Name": "476904687ba6567e465af6a0b34af147f6c714e389389627918c09872a1ad5d4",
                "Source": "/var/lib/docker/volumes/476904687ba6567e465af6a0b34af147f6c714e389389627918c09872a1ad5d4/_data",
                "Destination": "/opt/hgcapp/services-hedera/HapiApp2.0/data/upgrade",
                "Driver": "local",
                "Mode": "",
                "RW": true,
                "Propagation": ""
            },
            {
                "Type": "bind",
                "Source": "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/hedera.key",
                "Destination": "/opt/hgcapp/services-hedera/HapiApp2.0/hedera.key",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/data/onboard",
                "Destination": "/opt/hgcapp/services-hedera/HapiApp2.0/data/onboard",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/config.txt",
                "Destination": "/opt/hgcapp/services-hedera/HapiApp2.0/config.txt",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/network-logs/node/accountBalances",
                "Destination": "/opt/hgcapp/accountBalances",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node/compose-network/network-node/settings.txt",
                "Destination": "/opt/hgcapp/services-hedera/HapiApp2.0/settings.txt",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            }
        ],
        "Config": {
            "Hostname": "0d73d5dee15f",
            "Domainname": "",
            "User": "root",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "ExposedPorts": {
                "50111/tcp": {},
                "50211/tcp": {},
                "50212/tcp": {}
            },
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "JAVA_HEAP_MIN=256m",
                "JAVA_HEAP_MAX=2g",
                "JAVA_OPTS=\"-XX:+UnlockExperimentalVMOptions -XX:+UseZGC -Xlog:gc*:gc.log\"",
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "LC_ALL=C.UTF-8",
                "DEBIAN_FRONTEND=noninteractive",
                "JAVA_VERSION=jdk-17.0.3+7",
                "JAVA_HOME=/usr/local/java/jdk-17.0.3+7"
            ],
            "Cmd": null,
            "Healthcheck": {
                "Test": [
                    "CMD",
                    "grep",
                    "ServicesMain - Now current platform status = ACTIVE",
                    "/opt/hgcapp/services-hedera/HapiApp2.0/output/hgcaa.log"
                ],
                "Interval": 30000000000,
                "Timeout": 10000000000,
                "StartPeriod": 10000000000,
                "Retries": 10
            },
            "Image": "gcr.io/hedera-registry/main-network-node:0.27.4",
            "Volumes": {
                "/opt/hgcapp/accountBalances": {},
                "/opt/hgcapp/eventsStreams": {},
                "/opt/hgcapp/recordStreams": {},
                "/opt/hgcapp/services-hedera/HapiApp2.0/config.txt": {},
                "/opt/hgcapp/services-hedera/HapiApp2.0/data/config": {},
                "/opt/hgcapp/services-hedera/HapiApp2.0/data/diskFs": {},
                "/opt/hgcapp/services-hedera/HapiApp2.0/data/keys": {},
                "/opt/hgcapp/services-hedera/HapiApp2.0/data/onboard": {},
                "/opt/hgcapp/services-hedera/HapiApp2.0/data/saved": {},
                "/opt/hgcapp/services-hedera/HapiApp2.0/data/stats": {},
                "/opt/hgcapp/services-hedera/HapiApp2.0/data/upgrade": {},
                "/opt/hgcapp/services-hedera/HapiApp2.0/hedera.crt": {},
                "/opt/hgcapp/services-hedera/HapiApp2.0/hedera.key": {},
                "/opt/hgcapp/services-hedera/HapiApp2.0/log4j2.xml": {},
                "/opt/hgcapp/services-hedera/HapiApp2.0/output": {},
                "/opt/hgcapp/services-hedera/HapiApp2.0/settings.txt": {}
            },
            "WorkingDir": "/opt/hgcapp",
            "Entrypoint": [
                "/opt/hgcapp/services-hedera/HapiApp2.0/entrypoint.sh"
            ],
            "OnBuild": null,
            "Labels": {
                "com.docker.compose.config-hash": "26f080ce63b8899aca5dd59669907e131976429371bca100922d9dc87c67392e",
                "com.docker.compose.container-number": "1",
                "com.docker.compose.oneoff": "False",
                "com.docker.compose.project": "hedera-local-node",
                "com.docker.compose.project.config_files": "docker-compose.yml",
                "com.docker.compose.project.working_dir": "/home/nikolay/Desktop/mirror-node-rest-setup/hedera-local-node",
                "com.docker.compose.service": "network-node",
                "com.docker.compose.version": "1.25.0"
            }
        },
        "NetworkSettings": {
            "Bridge": "",
            "SandboxID": "fed43f4aea7f1dba9dcbff6c8abe1c403533c7e6e25fea077a2f7abd6c2aa03b",
            "HairpinMode": false,
            "LinkLocalIPv6Address": "",
            "LinkLocalIPv6PrefixLen": 0,
            "Ports": {},
            "SandboxKey": "/var/run/docker/netns/fed43f4aea7f",
            "SecondaryIPAddresses": null,
            "SecondaryIPv6Addresses": null,
            "EndpointID": "",
            "Gateway": "",
            "GlobalIPv6Address": "",
            "GlobalIPv6PrefixLen": 0,
            "IPAddress": "",
            "IPPrefixLen": 0,
            "IPv6Gateway": "",
            "MacAddress": "",
            "Networks": {
                "network-node-bridge": {
                    "IPAMConfig": {
                        "IPv4Address": "172.27.0.3"
                    },
                    "Links": null,
                    "Aliases": [
                        "network-node",
                        "0d73d5dee15f"
                    ],
                    "NetworkID": "9363b98aaacd3bac7a31144badf4d94eb017b169c99db74461e336f098596ff4",
                    "EndpointID": "",
                    "Gateway": "",
                    "IPAddress": "",
                    "IPPrefixLen": 0,
                    "IPv6Gateway": "",
                    "GlobalIPv6Address": "",
                    "GlobalIPv6PrefixLen": 0,
                    "MacAddress": "",
                    "DriverOpts": null
                }
            }
        }
    }
]

```

Comment (nathanklick) [reply .env:13]: @natanasow I was able to replicate the issue you have above. Your issue is caused by your installed version of the `docker-compose` software. Your installed version is `v1.25.0` however we require at least `v1.28.0` and at a maximum `v1.29.2` at this time. 

Please update your `docker-compose` to a supported version in the range of `>= 1.28.0 && <= 1.29.2`, here are some download links for Linux x86_64 for versions I have tested: 
- **v1.28.6** - https://github.com/docker/compose/releases/download/1.28.6/docker-compose-Linux-x86_64
- **v1.29.2** - https://github.com/docker/compose/releases/download/1.29.2/docker-compose-Linux-x86_64

Comment (nathanklick) [reply .env:13]: @natanasow Let's revert this change because the reported issue is due to an unsupported version of the `docker-compose` binary.

Comment (natanasow) [reply .env]: You were right, updating the `docker-compose` version fixed the errors.

**Review by nathanklick — APPROVED** [review]

**Pull request merged**